bitrise.yml

```diff
@@ -12,6 +12,11 @@ trigger_map:
 - tag: "*"
   workflow: release
 
+meta:
+  bitrise.io:
+    stack: osx-xcode-13.0.x
```

Comment:
Since we use the same stack for all workflows, it would be easier to have just one